src/main/java/ml/compute/InputData.java

```diff
@@ -0,0 +1,28 @@
+package ml.compute;
+
+import java.util.Set;
+
+import ml.preprocessor.types.PreprocessedData;
+import ml.preprocessor.types.RawData;
+
+/**
+ * Created by mikeecb on 17/10/2015.
+ */
+public class InputData {
```

Comment:
How about changing InputData to be a interface that has a function that returns X (which we still need to decide on) that openSMILE lot can implement for PreprocessedData and which we can do for RawData?